### PR TITLE
Makes rcd not work on reinforced doors.

### PR DESCRIPTION
**Pull request by AnturK**
:cl:
balance: Reinforced doors reinforcement need to be removed before RCD can deconstruct them.
/:cl:

This is not really a big change since there's almost certainly walls around the door.

### CODE REVIEW

**Comment by Fikou** [comment]
based malf ai buff

**Comment by yoyobatty** [comment]
At least it's not so bad, all you gotta do is remove the shielding, perfect rebalance.

**Comment by SouDescolado** [comment]
Will this also block admin RCD?

https://github.com/tgstation/tgstation/blob/a39d4ea60210af1f17f5bb9ec1ef6372b2a89cb6/code/game/objects/items/RCD.dm#L610

**Comment by MrDoomBringer** [comment]
This shouldn't block combat rcd

**Comment by zxaber** [comment]
> based malf ai buff

Presumably, the AI's shutters effectively accomplished this already, right?

**Comment by Daxxed** [comment]
Guess who died to an RCD
Jokes aside, this is good balance

**Comment by Barhandar** [comment]
This will prevent me from laughing at idiots who go through hacking upload airlocks normally en-masse like in recent rev round instead of getting a RCD, DOWNVOTED

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1402,6 +1402,9 @@
 /obj/machinery/door/airlock/rcd_vals(mob/user, obj/item/construction/rcd/the_rcd)
 	switch(the_rcd.mode)
 		if(RCD_DECONSTRUCT)
+			if(security_level != AIRLOCK_SECURITY_NONE)
+				to_chat(user,"<span class='notice'>[src] reinforcement needs to be removed first.</span>")
```
Comment (SpaceManiac) [inline code/game/machinery/doors/airlock.dm]:
```suggestion
				to_chat(user, "<span class='notice'>[src]'s reinforcement needs to be removed first.</span>")
```

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Comment by Fikou** [comment]
> > based malf ai buff
> 
> Presumably, the AI's shutters effectively accomplished this already, right?

yes but the shutters are destroyable with weaponry, the doors im pretty sure are invincible without explosives, so this will make the crew need to hack them or break the walls

**Review by SpaceManiac — APPROVED** [review]

**Pull request merged**